lib/supply/commands_generator.rb

```diff
@@ -19,8 +19,8 @@ def self.start
 
     def run
       program :version, Supply::VERSION
-      program :description, 'CLI for \'supply\' - TODO'
-      program :help, 'Author', 'Felix Krause <supply@krausefx.com>, Reinhard Hafenscher <TODO>'
+      program :description, Supply::DESCRIPTION
+      program :help, 'Author', 'Felix Krause <supply@krausefx.com>'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.